rich/logging.py

```diff
@@ -168,8 +169,13 @@ def render_message(self, record: LogRecord, message: str) -> "ConsoleRenderable"
             getattr(record, "markup") if hasattr(record, "markup") else self.markup
         )
         message_text = Text.from_markup(message) if use_markup else Text(message)
-        if self.highlighter:
-            message_text = self.highlighter(message_text)
+        highlighter = (
```

Comment:
How about this?

```python
highlighter = getattr(record, "highlighter", self.highlighter)
```

Reply:
Oh yes, that's neater -- I'll do the same for markup?